lib/msal-browser/docs/logging.md

```diff
@@ -0,0 +1,52 @@
+# MSAL JS logging instructions
+
+These are steps needed to enable MSAL JS logging for your applications:
+
+1. In the MSAL [Configuration](./configuration.md) object, you can enable logging to collect msal js logs. We enable different levels of logging and an appropriate level can be chosen as needed. 
+
+2. The logger options can be set as below, the example chooses `LogLevel.Trace`
+
+```javascript
+
+    ...
+    ...
+
+        system: {
+        loggerOptions: {
+            logLevel: msal.LogLevel.Trace,
```

Comment:
nit: Most people should be consuming the lib from npm where we don't use the `msal` namespace. Can we remove it from the example here and show the proper imports at the top?

Reply:
@derisen
Can you address this?

Reply:
Yeah probably the better approach for clarity and as default. Folks who are not consuming it from npm shouldn't have much trouble figuring it out